actors/multisig/src/lib.rs

```diff
@@ -306,6 +306,8 @@ impl Actor {
             )
         })?;
 
+        let resolved_new_signer = Address::new_id(resolved_new_signer);
```

Comment:
Change `is_signer` to take an ActorID instead, then just inline this `new_id` call when pushing onto st.signers.